public/react/SettingsSidebar/components/IssueSource/components/JqlTextArea/JqlTextArea.tsx

```diff
@@ -0,0 +1,55 @@
+import type { FC } from "react";
+
+import { Label } from "@atlaskit/form";
+import VisuallyHidden from "@atlaskit/visually-hidden";
+import React from "react";
+import TextArea from "@atlaskit/textarea";
+
+interface JQLTextAreaProps {
+  jql: string;
+  setJql: (jql: string) => void;
+  isLoading: boolean;
+  isSuccess: boolean;
+  totalChunks: number;
+  receivedChunks: number;
+  numberOfIssues: number;
+}
+
+const JQLTextArea: FC<JQLTextAreaProps> = ({
+  jql,
+  setJql,
+  isLoading,
+  isSuccess,
+  receivedChunks,
+  totalChunks,
+  numberOfIssues,
+}) => {
+  return (
+    <div className="flex items-end flex-col">
+      <VisuallyHidden>
+        <Label htmlFor="jql-text-area">Add your JQL</Label>
+      </VisuallyHidden>
+      <TextArea
+        id="jql-text-area"
```

Comment:
If there is a case where more than one instance of this component will exist in the DOM, maybe we should use a unique id instead